csrc/codegen.cpp

```diff
@@ -562,6 +562,21 @@ class CudaKernelGenerator : private kir::ConstIrVisitor {
     indent() << genCall(name, template_args, func_args) << ";\n";
   }
 
+  void genCpAsyncBulkTensorTile(const LoadStoreOp* ldst) {
+    auto in = ldst->in()->as<kir::TensorIndex>();
+    auto out = ldst->out()->as<kir::TensorIndex>();
+    TORCH_INTERNAL_ASSERT(
```

Comment:
Looks like there are still some uses of the old macros.

Reply:
Linter is green 😮

Reply:
Haha, it is red now with https://github.com/NVIDIA/Fuser/pull/894

Reply:
fixed